spec/features/manage_collections/open_drafts_spec.rb

```diff
@@ -75,7 +75,6 @@
           within '.eui-breadcrumbs' do
             expect(page).to have_content('Collection Drafts')
           end
-          expect(page).to have_content('Entry Title Not Provided')
         end
```

Comment:
For here, I think you should expect something, do a screenshot and look for some text in the JSON to verify the record loaded.

Reply:
expect(page).to have_content('Warning: Your Collection Draft has missing or invalid fields.')